infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -204,7 +213,7 @@ public int getConflictHashCode(UUID streamId, int conflictParam) {
      *     cause.
      */
     public TokenType txnCanCommit(TxResolutionInfo txInfo, /** Input. */
```

Comment:
Yeah sure, not in the scope of the PR (and not my code) but we can address that in another PR.